### PR TITLE
Reshuffle new payloads and schemas

### PR DESCRIPTION
**Pull request by itamargiv**
This is a followup to PR #86 . In this PR I make persisted schemas rely on interfaces for new variants so that the schmas won't diverge.

### CODE REVIEW

File: specs/global/schemas.json [hunk 1]
```diff
@@ -139,31 +139,24 @@
...
-            "qualifiers-order": {
-                "type": "array",
-                "items": {
-                    "type": "string"
+        "allOf": [
+            { "$ref": "./interfaces.json#/NewStatement" },
+            {
+                "type": "object",
+                "properties": {
+                    "id": {
```
Comment (wiese) [inline specs/global/schemas.json:147]: FTR this is comparable to a similar dilemma we faced in the TypeScript world, and solved it through optional properties there:
https://github.com/wmde/WikibaseDataModelTypes/blob/dd29f9b69b832a21618212fb23fd93b4df4f01d1/index.d.ts#L41

File: specs/global/interfaces.json [hunk 2]
```diff
@@ -108,7 +108,7 @@
         "type": "object",
         "properties": {
             "snaks": {
-                "$ref": "#/SnakMap"
+                "$ref": "#/Snaks"
```
Comment (wiese) [inline specs/global/interfaces.json:111]: I _think_ these will go away with a rebase vs #93 (in 363e588 & e966fb3 respectively).

**Review by wiese — APPROVED** [review]
Fine with me once the (rebase only) comment is resolved.

Thanks!

File: specs/global/schemas.json (hunk 1 above)
Comment (itamargiv) [reply specs/global/schemas.json:147]: Yeah, I wish there was a clearer option like that in OpenAPI Specs...

File: specs/global/interfaces.json (hunk 2 above)
Comment (itamargiv) [reply specs/global/interfaces.json:107]: :+1: Thanks, I didn't notice that. Rebased...

**Pull request merged**